src/reducers/auth.reducer.js

```diff
@@ -4,20 +4,26 @@ let user = JSON.parse(localStorage.getItem("user"));
 const initialState = user ? { loggedIn: true, user } : {};
 
 export function authentication(state = initialState,  { type, user }) {
-  switch (type) {
-    case userConstants.LOGIN_REQUEST:
-      return {
-        loggingIn: true,
-        user: user
-      };
-    case userConstants.LOGIN_SUCCESS:
-      return {
-        loggedIn: true,
-        user: user
-      };
-    case userConstants.LOGIN_FAILURE:
-      return {};
-    default:
-      return state
-  }
+    switch (type) {
+        case userConstants.LOGIN_REQUEST:
+            return {
+                loggingIn: true,
+                user: user
+            };
+        case userConstants.LOGIN_SUCCESS:
+            return {
+                loggedIn: true,
+                user: user
+            };
```

Comment:
```suggestion
        case userConstants.LOGIN_REQUEST:
        case userConstants.LOGIN_SUCCESS:
            return {
                loggedIn: true,
                user: user
            };
```
A shortcut to repeated the same code in more than one case